nuxt/nuxt.config.ts

```diff
@@ -189,4 +193,10 @@ export default defineNuxtConfig({
       background_color: '#202020',
     },
   },
+  turnstile: {
+    addValidateEndpoint: true,
```

Comment:
Are we using that endpoint? I think we're using [our own](https://github.com/maevsi/maevsi/pull/1122/files#diff-5f62f02e0d1371655f44eb36b384527c0d75ac421fe4249654b285ac607ce180) so this property can be removed.

Reply:
good point. changed that in https://github.com/maevsi/maevsi/pull/1122/commits/fe542d9e4ca65ea9c345d7fdfaa1a456dbabb5c7